src/desktop/components/main_layout/header/templates/index.jade

```diff
@@ -1,12 +1,4 @@
 include ../../../util/activator
 
-if sd.ENABLE_NEW_NAVBAR
-  header
-    != stitch.components.NavBar({ notificationCount: sd.NOTIFICATION_COUNT, searchQuery: sd.searchQuery, user: sd.CURRENT_USER,  })
-else
-  header#main-layout-header( class= sd.HEADER_CLASS ? sd.HEADER_CLASS : '' )
-    if sd.APPLICATION_NAME === "force-staging"
```

Comment:
I'm ok with dropping this staging banner (I never really understood it?).

Reply:
Staging banner is now in a  different component